test/Table_Tests/src/Common_Table_Operations/Take_Drop_Spec.enso

```diff
@@ -12,20 +12,24 @@ import Standard.Test.Extensions
 from project.Util import all
 from project.Common_Table_Operations.Util import run_default_backend
 
-main = run_default_backend spec
+import Standard.Database.Data.Column.Column as DB_Column
+import Standard.Database.Data.Table.Table as DB_Table
+import Standard.Test.Test_Result.Test_Result
 
 spec setup =
     prefix = setup.prefix
     table_builder = setup.table_builder
     test_selection = setup.test_selection
+    db_take_drop_by_pending = if setup.is_database.not then Nothing else "Database backends do not support Index_Sub_Range.While or .Sample."
     take_drop_by_pending = if test_selection.take_drop.not then
```

Comment:
Should we remove the take_drop backend config entirely? I imagine it will be implementable for all database backends.

Reply:
Yeah, I think we can delete it. We can always re-add it if needed.

Reply:
Done